src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  padding: 0 50px;
```

Comment:
Ensure that the padding values are consistent with the design. If the design specifies different padding values for different screen sizes, consider using media queries to adjust the padding accordingly.